tests/Calculator/CalculatorTestCase.php

```diff
@@ -198,4 +207,14 @@ public function compareExamples()
             ['1', '0.000000000000000000000000005', 1],
         ];
     }
+
+    public function modExamples()
+    {
+        return [
```

Comment:
Can you please add a few tests for negative numbers as well?

Reply:
Good spot, I've added some and they flagged up a bug in the `GmpCalculator::mod()` implementation, which I've now fixed.